pkg/apis/kubeone/types.go

```diff
@@ -439,7 +439,10 @@ type HetznerSpec struct {
 }
 
 // KubevirtSpec defines the Kubevirt provider
-type KubevirtSpec struct{}
+type KubevirtSpec struct {
+	// InfraNamespace
```

Comment:
Can you please add a proper godoc-based description for this field? This description is used for the user-facing documentation.